public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -240,12 +239,28 @@ function ModalMissionComplete (svl, missionContainer, missionModel, taskContaine
         var unit = {units: 'miles'};
         var regionId = neighborhood.getProperty("regionId");
 
+        console.log("User has finished a particular mission...");
+
         var missionDistance = mission.getDistance("miles");
+        missionDistance = Math.round(missionDistance * 10) / 10;
```

Comment:
I have two comments for these. The first is, like you said, it would be cleaner for the code to looks like
```
var missionDistance = mission.getDistance("miles").toFixed(1);
```

The second point is that you are rounding numbers that are then used in later calculations, which can cause the later numbers to more inaccurate. I think that what you want to do is round these numbers only when you actually use them. Like on develop right now, the `missionDistance` is not rounded until we actually use it in `_updateMissionProgressStatistics` function. So if the problem is that things should be rounded in the `modalMissionProgressBar.update`, just round them in that function and not earlier.